reporting/api/src/main/java/org/jboss/windup/reporting/service/MapSumEffortAccumulatorFunction.java

```diff
@@ -0,0 +1,44 @@
+package org.jboss.windup.reporting.service;
+
+
+import com.tinkerpop.blueprints.Vertex;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
```

Comment:
Fixed:
 * This implementation holds a Map, and for each vertex, it increases the key
 * that is determined from the vertex by supplied implementation of vertexToKey({@link Vertex}).